debian/changelog

```diff
@@ -1,3 +1,9 @@
+open-build-service (2.7.4-2endless3) UNRELEASED; urgency=medium
+
+  * Pin the version of minitest to that packaged by Debian.
+
+ -- Nimrod Adar <nimrod@endlessm.com>  Tue, 14 May 2019 12:16:13 +0300
```

Comment:
For future reference, you don't actually have to add changelog entries to our CI managed packages except in special circumstances where you want a new version like an upstream release. Typically, though, we just let the jenkins tooling make up versions and changelog entries on the fly. https://phabricator.endlessm.com/w/software/build/ci-integration/ describes it a bit, although that's for the OS packages.